src/SDPoptimize.jl

```diff
@@ -552,6 +555,7 @@ the alea realization
 
 Returns :
 - V(x0) (Float64)
+
 """
 function get_control(model::SPModel,param::SDPparameters,V::Array{Float64}, t::Int64, x::Array)
 
```

Comment:
We should check if we are in the DH case, if not warn("the problem is defined in hazard-decision and control is obtained assuming decision-hazard structure")